add/metadata/System.Net/HttpWebResponse.meta.md

```diff
@@ -65,7 +65,7 @@ manager: "markl"
 ---
 
 ---
-uid: System.Net.HttpWebResponse.Dispose
+uid: System.Net.HttpWebResponse.Dispose*
```

Comment:
this is different. the uids are usually for the specific APIs, not for the overload. should we undo these changes?

see example here: https://raw.githubusercontent.com/dotnet/docs/master/add/metadata/System.IO/StreamReader.meta.md

Reply:
Hi Maira, 

Please check this page: [Dispose](https://www.microsoftonedoc.com/#/organizations/e6f6a65cf14f462597b64ac058dbe1d0/projects/3fedad16-eaf1-41a6-8f96-0c1949c68f32/containers/bcddac22-afb3-4f80-9ba0-ebbea45c68fa/tocpaths/a8530c58-2ee6-4e2c-972f-011154f8c87a/locales/en-US).

This item should be used for overload. We already have an item with UID: System.Net.HttpWebResponse.Dispose(System.Boolean) in this metadata file.

Reply:
Sure. My point was just that I haven't seen the * being used in metadata files before.